interactions/client/client.py

```diff
@@ -2002,6 +2004,36 @@ def load_extension(
         module = importlib.import_module(module_name, package)
         self.__load_module(module, module_name, **load_kwargs)
 
+    def load_extensions(
+        self,
+        *packages: str,
+        recursive: bool = False,
+    ) -> None:
+        """
+        Load multiple extensions at once.
+
+        Removes the need of manually looping through the package
+        and loading the extensions.
+
+        Args:
+            *packages: The package(s) where the extensions are located.
+            recursive: Whether to load extensions from the subdirectories within the package.
+        """
+        if not packages:
+            raise ValueError("You must specify at least one package.")
+
+        for package in packages:
+            # If recursive then include subdirectories ('**')
+            # otherwise just the package specified by the user.
+            pattern = os.path.join(package, "**" if recursive else "", "*.py")
+
+            # Find all files matching the pattern, and convert slashes to dots.
+            extensions = [f.replace(os.path.sep, ".").replace(".py", "") for f in glob.glob(pattern, recursive=True)]
+
+            for ext in extensions:
+                print(ext)
```

Comment:
Either remove this, or change it to a nicer message sent through the logger.

Reply:
My testing print statement, thanks for pointing that out.